annotation_pipeline/image.py

```diff
@@ -99,17 +160,16 @@ def process_centr_segment(obj, id, ibm_cos):
         sp_arr = read_ds_segments(ds_bucket, ds_segm_prefix, first_ds_segm_i, last_ds_segm_i, ibm_cos)
 
         formula_images_it = gen_iso_images(sp_inds=sp_arr[:,0], sp_mzs=sp_arr[:,1], sp_ints=sp_arr[:,2],
-                                           centr_df=centr_df,
+                                           centr_f_inds=centr_df.formula_i.values, centr_p_inds=centr_df.peak_i.values,
+                                           centr_mzs=centr_df.mz.values, centr_ints=centr_df.int.values,
```

Comment:
Please pass just `centr_df` here, and expand these 4 variables inside `gen_iso_images`. It makes it a bit cleaner, as the complexity is closer to the code that depends on it, and it makes it easier to see where the `centr_f_inds` abbreviation comes from.

Reply:
changed